plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/fixtures/attributes.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Attributes data
```

Comment:
Was this line left here intentionally? I'm asking because the explanation below seems more complete. If we plan to leave this line, I would add a period (`.`) at the end.

Reply:
This is like a title :-) Going to clean it in a follow-up